README.md

```diff
@@ -1,16 +1,11 @@
 # OMEZarrReader
+[![Actions Status](https://github.com/ome/ZarrReader/workflows/Maven/badge.svg)](https://github.com/ome/ZarrReader/actions)
 
 The OMEZarrReader provides support for reading data following the [NGFF](https://ngff.openmicroscopy.org/)
 specificaiton. It uses the [JZarr](https://github.com/zarr-developers/jzarr) library from [Maven Central](https://central.sonatype.com/artifact/dev.zarr/jzarr/) for accessing the underlying Zarr data.
 
 ## Installation
 
-### Fiji Update Site
-
-The OMEZarrReader has been added to the Bio-Formats development update site
```

Comment:
Is this no longer the case?

Reply:
I cannot see any zarr Reader on the update site